src/main/java/org/prebid/server/proto/openrtb/ext/response/Cache.java

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.proto.openrtb.ext.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+
+@AllArgsConstructor(staticName = "of")
+@Builder
+@Value
+public class Cache {
```

Comment:
i think we can use `org.prebid.server.proto.openrtb.ext.response.ExtResponseCache` instead of class - just change it for our case.